examples/select_datasets.md

```diff
@@ -20,24 +20,25 @@
 
 `client.select_datasets(where='cluster', ...)`:
 ```python
->>> cluster_name = client.select_clusters().get_list()[30]['grouping_name']
```

Comment:
Do unconstrained searches no longer work? Is there an expected error behavior if a user attempts an unconstrained search?

Reply:
Not all clusters are guaranteed to be good input to cluster->dataset queries.  If a cluster isn't, the query should return, but the assert statement at the end of this block will fail.